src/main/java/com/thevoxelbox/voxelsniper/brush/CanyonBrush.java

```diff
@@ -19,7 +19,8 @@
  */
 public class CanyonBrush extends Brush {
 
-    private static final int SHIFT_LEVEL_MIN = 10;
+    //This is stupid, but i'm also lazy
+    private static final int SHIFT_LEVEL_MIN = -64+10;
```

Comment:
```suggestion
    private static final int SHIFT_LEVEL_MIN = 10;
```

This seems to be changing behaviour. This causes the for loop at line 61 to not run at all, since `this.getMinHeight()+SHIFT_LEVEL_MIN` would always be lower then `this.getMinHeight()+1` when SHIFT_LEVEL_MIN is -54.